### DISCUSSION TITLE
Pin station on Band Map

### DISCUSSION THREAD

**Discussion by vazquezrodjoel** [foldynl/QLog#512] [Polls]
Would it be beneficial to add the ability of pin a station callsign on the band map from the New Contact window/section.  For example, right click on the callsign field and "Pin to Band Map".  That way station that you cant work at that particular moment you can easily come back to it to give another try, like dx-peditions etc...

**Comment by foldynl** [comment]
To be honest, I don't understand what it is supposed to do. So, if I have a callsign pinned, does that mean the bandmap focuses on that specific DX on the given band and remains focused on that DX? Should this really be a function of the bandmap?

**Comment by aa5sh** [comment]
Could it be something like in N1MM where if you type in a callsign and spin
the dial it add that like a spot entry?

**Comment by vazquezrodjoel** [comment]
For example, I copy a station and it still hasnt pop up on the cluster yet (or even I dont have a cluster conection open), I cant work that station right now, but I dont want to forget where it was so I can try again later, so I "Pin it" on the bandmap (so I  just manually add the callsign to the bandmap with just a click) so I can later come back to that frequency just clicking on the callsign from the bandmap.  

Regarding been automatic, I dont think that would be good though, but not sure.  I would like the capacity to manually add it to the bandmap.
